### ISSUE TITLE
Modify Alt Text for Banner

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#2674]
Hyku version: 6.1.0.0de435e845
Knapsack version: 0.0.1.5069cc809d

### Descriptive summary

Banners can be decorative or not. If decorative, the alt attribute should be empty. If not, descriptive text should inserted. In Hyku, descriptive text is the only option.

### Steps to reproduce the behavior in User Interface (UI)

1. Dashboard
2. Settings
3. Appearance
4. Banner Image tab

<img width="800" height="898" alt="Image" src="https://github.com/user-attachments/assets/86bcad08-a5d1-4aa2-8b9f-cf3e9b515963" />

### Actual behavior (include screenshots if available)

The user should choose which type of image this is - decorative or not. If decorative, the alt attribute should be empty. If not, minimally a string should be required before saving.

### Acceptance Criteria/Expected Behavior

- [ ] User should identify a banner image as decorative, provide the image and be able to save successfully.
- [ ] If not decorative, the user should be required to provide a string at minimum to describe the image.

### Rationale (for feature request only)

Accessibility guidelines require that all images have an alt attribute. If the image is decorative only and provides no information, the alt attribute should be empty. If not, patrons using screen readers will hear the filename as the alternative text.

Section 508 guidance: https://www.section508.gov/create/alternative-text/#decorative-images
Decorative Images https://www.w3.org/WAI/tutorials/images/decorative/
An alt Decision Tree https://www.w3.org/WAI/tutorials/images/decision-tree/
Using null alt text and no title attribute on img elements for images that assistive technology should ignore https://www.w3.org/WAI/WCAG22/Techniques/html/H67
Failure of marking up decorative images . . . https://www.w3.org/WAI/WCAG22/Techniques/failures/F38

### Related work

Link to related issues or prior related work here.
